hopp/simulation/technologies/dispatch/hybrid_dispatch_builder_solver.py

```diff
@@ -674,12 +674,12 @@ def simulate_with_dispatch(
 
     def battery_heuristic(self):
         tot_gen = [0.0] * self.options.n_look_ahead_periods
```

Comment:
Could this be simplified to the following?

```python
tot_gen = np.zeros(self.options.n_look_ahead_periods)
for power_source in self.power_sources.keys():
    if "battery" in power_source or "grid" in power_source:
        continue
    tot_gen += self.power_sources[power_source].dispatch.available_generation
```

Reply:
Yes and done

Reply:
I think it needs to get pushed, but I'll trust it's on its way.

Reply:
Good catch. I'm not sure lumpy will work without having knock-on effects, but I'll try it